apps/src/applab/designMode.js

```diff
@@ -650,6 +656,30 @@ function makeUndraggable(jqueryElements) {
   return foundOne;
 }
 
+/**
+ * Highlights an element with a dashed border, removes border from all other elements
```

Comment:
How about adding something like "must only be called on an element wrapped in a draggable div" to the comments (or instead checking that it's parent is draggable if you prefer).